components/clickToCopy.js

```diff
@@ -0,0 +1,64 @@
+import copy from 'copy-to-clipboard';
+import Button from './button';
+
+export default class ClickToCopy extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      copied: false
+    };
+  }
+  copyNow(t) {
+    copy(t);
+    this.setState({
+      copied: true
+    });
+  }
+  componentWillReceiveProps() {
+    this.setState({
+      copied: false
+    });
+  }
+  render() {
+    const { target, children } = this.props;
+    const { copied } = this.state;
+    return (
+      <span className={'copyElem ' + copied}>
+        <span onClick={() => this.copyNow(target)}>
+          <Button text={copied ? 'copied 👌' : 'copy'} />
+        </span>
+        {children}
+        <style jsx>{`
+            .copyBtn {
```

Comment:
I should have removed those. 

But I will still leave line 57-59 here. It indicates the user the content is been copied.